test.js

```diff
@@ -0,0 +1,38 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+var TestUtils = require('react-addons-test-utils');
+var OnClickOutside = require('.');
+
+describe('react-onclickoutside', function() {
+
+  var OutsideClickComponent = React.createClass({
+    mixins: [OnClickOutside],
+
+    getInitialState: function() {
+      return {handleClickOutsideCalled: false};
+    },
+
+    handleClickOutside: function() {
+      this.setState({handleClickOutsideCalled: true});
+    },
+
+    render: function() {
+      return React.createElement('div');
+    }
+  });
+
+  it('should not call handleClickOutside when clicking the element', function() {
+    var element = React.createElement(OutsideClickComponent);
+    var component = TestUtils.renderIntoDocument(element);
+    ReactDOM.findDOMNode(component).dispatchEvent(new MouseEvent('mousedown'));
+    assert(!component.state.handleClickOutsideCalled);
+  });
+
+  it('should call handleClickOutside when clicking the document', function() {
+    var element = React.createElement(OutsideClickComponent);
+    var component = TestUtils.renderIntoDocument(element);
+    document.dispatchEvent(new MouseEvent('mousedown'));
```

Comment:
mostly "not tied to any specific browser". I'm a big proponent of "it has to work on the internet" rather than "it has to work in browser X" so if this only works in Chrome, that's the the wrong way to go about it really... =/

The tests should pass irrespective of which browser's being used, as long as it conforms to modern APIs.